### PR TITLE
report MemberID on registerMember

### PR DESCRIPTION
**Pull request by sd00070**
* create `RegisterMember` stored procedure
* attach stored procedure to DAL
* pass insert ID from DAL to controller to view
* display MemberID in MessageBox

### CODE REVIEW

**Comment by sd00070** [comment]
Once this branch is pulled, you all will need to run the RegisterMember.sql script to add the stored procedure to your db.

File: RentMe-App/DAL/MemberDBDAL.cs [hunk 1]
```diff
@@ -178,22 +178,19 @@ public List<Member> GetMemberByPhone(string phone)
         /// </summary>
```
Comment (kmarroq1) [inline RentMe-App/DAL/MemberDBDAL.cs:178]: Clean this file.

**Review by kmarroq1 — CHANGES_REQUESTED** [review]
Everything looks good! Just clean up the dal, then should be good to be merged!

**Comment by sd00070** [comment]
merge should be ready

**Review by kmarroq1 — APPROVED** [review]

**Pull request merged**